modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -97,6 +100,8 @@ public class EmailNotificationTypeTest extends BaseNotificationTypeTest {
 	public static void setUpClass() throws Exception {
 		BaseNotificationTypeTest.setUpClass();
 
+		_siteGroup = GroupTestUtil.addGroup();
```

Comment:
Hey @rijema rename `_siteGroup` to `_group` because we don't have two variables of the type Group in this class that could have a name conflict.

Thank you!